### PR TITLE
License the project under the terms of the GPL-3.0-or-later

### PR DESCRIPTION
**Pull request by dvzrv**
Add a LICENSE file (GPL-3.0) and add a license statement to the project's README file.
Replace the use of "unspecified" license header in all scripts with the SPDX identifier for GPL-3.0-or-later.
Replace the use of GPL-2.0 license header in all scripts with the SPDX identifier for GPL-3.0-or-later.

This change can only be merged if all of the below authors consent to licensing/relicensing their work under the terms of the GPL-3.0-or-later or if their contributions are removed. The authors of the project have been derived using `git log --all --format='%aN <%aE>' |sort -u`.

```
Aaron Griffin <aaronmgriffin@gmail.com>
Alad Wenter <alad@archlinux.info>
Allan McRae <allan@archlinux.org>
Andrea Scarpino <andrea@archlinux.org>
Armin Luntzer <armin@archlinuxppc.org>
Bartłomiej Piotrowski <b@bpiotrowski.pl>
Bartłomiej Piotrowski <bpiotrowski@archlinux.org>
Biru Ionut <biru.ionut@gmail.com>
Biru Ionut <ionut@archlinux.ro>
BlackEagle <ike.devolder@gmail.com>
Brian Bidulock <bidulock@openss7.org>
Byron Clark <byron@theclarkfamily.name>
Christian Hesse <mail@eworm.de>
Damir Perisa <damir.perisa@solnet.ch>
Dan McGee <dan@archlinux.org>
Daniel Isenmann <daniel@archlinux.org>
Daniel J Griffiths <ghost1227@archlinux.us>
Daniel M. Capella <polyzen@archlinux.org>
Dave Reisner <d@falconindy.com>
Dave Reisner <dreisner@archlinux.org>
David Runge <dvzrv@archlinux.org>
Doug Newgard <scimmia@archlinux.info>
Eli Schwartz <eschwartz@archlinux.org>
Eli Schwartz via arch-projects <arch-projects@archlinux.org>
Emiel Wiedijk <me@aimileus.nl>
Eric Bélanger <eric@archlinux.org>
Eric Bélanger <snowmaniscool@gmail.com>
Eric Blanger <snowmaniscool@gmail.com>
Erich Eckner <git@eckner.net>
Evangelos Foutras <evangelos@foutrelis.com>
Evangelos Foutras <foutrelis@gmail.com>
Felix Yan <felixonmars@archlinux.org>
Florian Pritz <bluewind@server-speed.net>
Florian Pritz <bluewind@xinu.at>
Florian Pritz <bluewind@xssn.at>
Francois Charette <francois@archlinux.org>
Francois Charette <francois@samarqand.localdomain>
François Charette <francois@archlinux.org>
Gerardo Exequiel Pozzi <vmlinuz386@yahoo.com.ar>
Ionut Biru <ibiru@archlinux.org>
Ionut Biru <ionut@archlinux.ro>
Ivy Foster <escondida@iff.ink>
Jakob Gruber <jakob.gruber@gmail.com>
Jan Alexander Steffens (heftig) <jan.steffens@gmail.com>
Jan Steffens <jan.steffens@gmail.com>
Jason Chu <jason@archlinux.org>
Jason Chu <jchu@xentac.net>
Jelle van der Waa <jelle@archlinux.org>
Jelle van der Waa <jelle@vdwaa.nl>
Joel Teichroeb <joel@teichroeb.net>
Johannes Löthberg <johannes@kyriasis.com>
Konstantin Gizdov <kgizdov@gmail.com>
Laurent Carlier via arch-projects <arch-projects@archlinux.org>
Levente Polyak <anthraxx@archlinux.org>
Lukas Fleischer <archlinux@cryptocrack.de>
Luke Shumaker <LukeShu@sbcglobal.net>
Luke Shumaker <lukeshu@lukeshu.com>
Luke Shumaker <lukeshu@parabola.nu>
Markus M. May <mmay@javafreedom.org>
Martchus <martchus@gmx.net>
Matt Robinson <git@nerdoftheherd.com>
Maxime Gauduin <alucryd@gmail.com>
Morten Linderud <foxboron@archlinux.org>
Nezmer <Nezmer@allurelinux.org>
NicoHood <git@nicohood.de>
Pierre Schmitz <pierre@archlinux.de>
Ray Rashif <schivmeister@gmail.com>
Robin Broda <robin@broda.me>
Ronald van Haren <pressh@gmail.com>
Rémy Oudompheng <remy@archlinux.org>
Simo Leone <simo@archlinux.org>
Sven-Hendrik Haase <svenstaro@gmail.com>
Sébastien Luttringer <seblu@seblu.net>
Thomas Bächler <thomas@archlinux.org>
Timothy Redaelli <timothy.redaelli@gmail.com>
Tom Gundersen <teg@jklm.no>
Travis Willard <travis@archlinux.org>
WorMzy Tykashi <wormzy.tykashi@gmail.com>
bartoszek <bartoszek.github@bartus.33mail.com>
jelle van der Waa <jelle@archlinux.org>
lukeshu@parabola.nu <lukeshu@parabola.nu>
user <user@Nokia-N900.(none)>
```

For convenience all users with a known github account have been added to the below list of checkboxes:

- [x] @aaronmgriffin 
- [x] @AladW 
- [x] @allanmcrae 
- [x] @ilpianista 
- [x] @barthalion 
- [x] @ionutbiru 
- [x] @BlackIkeEagle 
- [x] @bbidulock 
- [x] @byronclark 
- [x] @eworm-de 
- [x] @toofishes 
- [x] @isenmann 
- [x] @polyzen 
- [x] @falconindy 
- [x] @dvzrv 
- [x] @eli-schwartz 
- [x] @aimileus
- [x] @EricBelanger 
- [x] @foutrelis 
- [x] @felixonmars 
- [x] @Bluewind 
- [x] @djgera 
- [x] @escondida
- [x] @schuay 
- [x] @heftig 
- [x] @jelly 
- [x] @klusark 
- [x] @kyrias 
- [x] @kgizdov 
- [x] @anthraxx 
- [x] @lfos 
- [x] @LukeShu 
- [x] @triplem 
- [x] @Martchus 
- [x] @ribbons 
- [x] @alucryd 
- [x] @Foxboron 
- [x] @NicoHood 
- [x] @pierres 
- [x] @schivmeister 
- [x] @coderobe 
- [x] @simoleone 
- [x] @svenstaro 
- [x] @seblu 
- [x] @drizzt (commit reverted)
- [x] @teg 
- [x] @WorMzy 
- [x] @bartoszek 

All people in the below list need to be contacted via mail:

- [x] `Armin Luntzer <armin@archlinuxppc.org>`
- [x] `Damir Perisa <damir.perisa@solnet.ch>`
- [x] `Daniel J Griffiths <ghost1227@archlinux.us>`
- [x] `Doug Newgard <scimmia@archlinux.info>`
- [x] `Erich Eckner <git@eckner.net>`
- [x] `Francois Charette <francois@archlinux.org>`/ `Francois Charette <francois@samarqand.localdomain>`/ `François Charette <francois@archlinux.org>`
- [x] `Jason Chu <jason@archlinux.org>`/ `Jason Chu <jchu@xentac.net>`
- [x] `Laurent Carlier via arch-projects <arch-projects@archlinux.org>`
- [x] `Nezmer <Nezmer@allurelinux.org>` (commits removed/ not in use anymore)
- [x] `Ronald van Haren <pressh@gmail.com>` (commits removed/ not in use anymore)
- [x] `Rémy Oudompheng <remy@archlinux.org>`
- [x] `Thomas Bächler <thomas@archlinux.org>`
- [x] `Travis Willard <travis@archlinux.org>`
- [x] Someone with a Nokia N900 (`user <user@Nokia-N900.(none)>`) who committed https://github.com/archlinux/devtools/commit/7e5ae10b43a3b74b3c3236c5a36616704e3c1747 (this appears to have been @ilpianista as mentioned in his comment https://github.com/archlinux/devtools/pull/62#issuecomment-858904225)

Implements #61

### CODE REVIEW

**Comment by dvzrv** [comment]
I consent to my work on devtools to be licensed under the terms of the GPL-3.0-or-later.

**Comment by Foxboron** [comment]
I consent to my work on devtools to be licensed under the terms of the GPL-3.0-or-later.

**Comment by heftig** [comment]
I consent to my work on devtools to be licensed under the terms of the GPL-3.0-or-later.

**Comment by svenstaro** [comment]
I consent to my work on devtools to be licensed under the terms of the GPL-3.0-or-later.

**Comment by klusark** [comment]
I consent to my work on devtools to be licensed under the terms of the GPL-3.0-or-later.

**Comment by polyzen** [comment]
I consent to my work on devtools to be licensed under the terms of the GPL-3.0-or-later.

BTW, archlinux.info is listed for scimmia's email, which afaik is still defunct.

--
Best,
Daniel <https://danielcapella.com>

**Comment by coderobe** [comment]
I consent to my work on devtools to be licensed under the terms of the GPL-3.0-or-later.

**Comment by felixonmars** [comment]
I consent to my work on devtools to be licensed under the terms of the GPL-3.0-or-later.

**Comment by dvzrv** [comment]
I have reached out to the people without github account assignments and will post their mail responses verbatim as a comment.

**Comment by dvzrv** [comment]
@seblu only few of the scripts in this repository actually had license statements (which is problematic).

**Comment by teg** [comment]
I consent to my work on devtools to be licensed under the terms of the GPL-3.0-or-later.

**Comment by aaronmgriffin** [comment]
I consent to all this jazz

**Comment by eworm-de** [comment]
I consent to my work on devtools to be licensed under the terms of the GPL-3.0-or-later.

**Comment by WorMzy** [comment]
I only have a single commit which no longer exists in the form I submitted it, so my consent is likely redundant, but in either case I consent to my work on devtools to be licensed under the terms of the GPL-3.0-or-later.

**Comment by deep-42-thought** [comment]
Erich Eckner: I consent to my work on devtools to be licensed under the terms of the GPL-3.0-or-later.

**Comment by ionutbiru** [comment]
I consent to my work on devtools to be licensed under the terms of the GPL-3.0-or-later.

> Acked-by: Biru Ionut <biru.ionut@gmail.com>
> Acked-by: Biru Ionut <ionut@archlinux.ro>

**Comment by escondida** [comment]
I consent to have my contributions to Archlinux's devtools licensed
under the GPL v. 3 or later.

Cheers,
Ivy Foster

**Comment by isenmann** [comment]
I consent to my work on devtools to be licensed under the terms of the GPL-3.0-or-later.

**Comment by alucryd** [comment]
I consent to my work on devtools to be licensed under the terms of the GPL-3.0-or-later.

**Comment by NicoHood** [comment]
Also okay for me

**Comment by Martchus** [comment]
I consent to my work on devtools to be licensed under the terms of the GPL-3.0-or-later.

**Comment by BlackIkeEagle** [comment]
I consent to my work on devtools to be licensed under the terms of the GPL-3.0-or-later.

**Comment by simoleone** [comment]
I consent to my work on devtools to be licensed under the terms of the GPL-3.0-or-later.

**Comment by jelly** [comment]
I consent to my work on devtools to be licensed under the terms of the GPL-3.0-or-later.

**Comment by aimileus** [comment]
I consent to my work on devtools to be licensed under the terms of the GPL-3.0-or-later.

**Comment by byronclark** [comment]
I consent to my work on devtools to be licensed under the terms of the GPL-3.0-or-later.

**Comment by kyrias** [comment]
I consent to my work on devtools to be licensed under the terms of the GPL-3.0-or-later.

**Comment by twillard** [comment]
It is I, <travis@archlinux.org>

I consent to my work on devtools to be licensed under the terms of the GPL-3.0-or-later.

**Comment by triplem** [comment]
This is Markus (@j..freedom.org).

I consent to my work on devtools to be licensed under the terms of the GPL-3.0-or-later.

**Comment by Bluewind** [comment]
I consent to my work on devtools to be licensed under the terms of the GPL-3.0-or-later.

**Comment by lfos** [comment]
I consent to my work on devtools to be licensed under the terms of the GPL-3.0-or-later.

**Comment by schuay** [comment]
I consent to my work on devtools to be licensed under the terms of the GPL-3.0-or-later.

**Comment by djgera** [comment]
I consent to my work on devtools to be licensed under the terms of the 
GPL-3.0-or-later.

**Comment by anthraxx** [comment]
gentle ping, please send a consent :cat: 
@allanmcrae
@barthalion
@bbidulock
@toofishes
@falconindy
@eli-schwartz
@EricBelanger
@LukeShu
@schivmeister
@pierres
@drizzt
@bartoszek
@Scimmia22

**Comment by eli-schwartz** [comment]
>  Replace the use of GPL-2.0 license header in all scripts with the SPDX identifier for GPL-3.0-or-later.

These are 50% mine. :p

All of my work on devtools to date is licensed under the terms of the  GPL-2.0-or-later, either by:
- pre-existing explicit license declaration or,
- via my original intent when contributing to the project which was partially licensed GPL-2.0-only and partially licensed under my GPL-2.0-or-later scripts,
- via this statement: I consent to my work on devtools to be licensed under the terms of the GPL-2.0-or-later.

The project is then of course free to use my work under the "or later" part of this to make a combined work only usable under the GPL-3.0-or-later when considering sourced/m4_included shell libraries or other people contributing GPL-3.0-or-later enhancements... last year's existing licenses cannot be rescinded though. :p

tl;dr from my end this is fine.

**Comment by pierres** [comment]
Sure, GPL is fone by me (thought it already was)

**Comment by dvzrv** [comment]
@Nezmer if you have contributed to this repository as `Nezmer <Nezmer@allurelinux.org>` please respond to this ticket :)

**Comment by toofishes** [comment]
I consent to my work on devtools to be licensed under the terms of the GPL-3.0-or-later.

**Comment by lordheavy** [comment]
I consent to my work on devtools to be licensed under the terms of the GPL-3.0-or-later.

**Comment by remyoudompheng** [comment]
I consent to my work on devtools to be licensed under the terms of the GPL-3.0-or-later.

This is for contributions under email address remy@archlinux.org

**Comment by allanmcrae** [comment]
I consent to my work on devtools to be licensed under the terms of the GPL-3.0-or-later.

**Comment by LukeShu** [comment]
I consent to my work on devtools to be licensed under the terms of the GPL-3.0-or-later.

**Comment by barthalion** [comment]
I consent to my work on devtools to be licensed under the terms of the GPL-3.0-or-later.

**Comment by yan12125** [comment]
Now that my small patch from https://github.com/archlinux/devtools/pull/56 is applied, here is my statement:

I consent to my work on devtools to be licensed under the terms of the GPL-3.0-or-later.

**Comment by dvzrv** [comment]
Another gentle ping towards the remaining people (who do have a known github account):

- @bbidulock 
- @falconindy 
- @EricBelanger 
- @schivmeister 
- @drizzt 
- @bartoszek 

Please consent to the relicensing of the devtools project :)

**Comment by schivmeister** [comment]
I consent to my work on devtools to be licensed under the terms of the GPL-3.0-or-later.

This is for contributions under any one of the following identities and email addresses:

Ray Rashif <schivmeister gmail com>
Rashif Rahman <schivmeister gmail com>
Rashif Ray Rahman <schiv archlinux org>

**Comment by falconindy** [comment]
I consent to my work on devtools to be licensed under the terms of the GPL-3.0-or-later.

**Comment by bartoszek** [comment]
I consent to my work on devtools to be licensed under the terms of the GPL-3.0-or-later.

This is for contributions under any one of the following identities and email addresses:

bartoszek <bartoszek.github@bartus.33mail.com>
bartus <szczepaniak.bartek+github@gmail.com>

**Comment by dvzrv** [comment]
Thomas Bächler agrees: 
[thomas_baechler.txt](https://github.com/archlinux/devtools/files/6873905/thomas_baechler.txt)

**Comment by mavelli** [comment]
I consent to my work on devtools to be licensed under the terms of the GPL-3.0-or-later.

**Comment by bbidulock** [comment]
I consent to my work on devtools to be licensed under the terms of the GPL-3.0-or-later.

**Comment by fc7** [comment]
I consent to my work on devtools to be licensed under the terms of the GPL-3.0-or-later.

François Charette (for the following commit identities:)
```
Francois Charette <francois@archlinux.org>
Francois Charette <francois@samarqand.localdomain>
François Charette <francois@archlinux.org>
```

**Comment by sudoforge** [comment]
I consent to my work on devtools to be licensed under the terms of the GPL-3.0-or-later.

```
Benjamin Denhartog <ben@sudoforge.com>
```

**Comment by evertiro** [comment]
I consent to my work on devtools being relicensed under the terms of the GPL-3.0 or later.

```
Daniel J Griffiths <ghost1227@archlinux.us>
```

**Comment by aluntzer** [comment]
Armin Luntzer: I consent to my work on devtools to be licensed under the terms of the GPL-3.0-or-later.

**Review by jelly — APPROVED** [review]

**Comment by bbidulock** [comment]
I consent to my work on devtools to be licensed under the terms of the GPL-3.0-or-later.

**Comment by drizzt** [comment]
I consent to my work on devtools to be licensed under the terms of the GPL-3.0-or-later.
